.github/workflows/pull_request.yml

```diff
@@ -38,19 +29,24 @@ jobs:
       - name: Lint SCSS
         run: yarn lint:scss
 
-  typescript:
-    name: Typescript compile check
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Setup Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: '12.22.0'
-          cache: 'yarn'
-      - name: Run typecript compile check
-        run: npx ts-compile-checker
-        working-directory: './src'
+  # This job is skipped because it started to fail
+  # An issue has been filed to get it fixed:
+  # https://github.com/matis-dk/ts-compile-checker/issues/1
+  # When it is fixed we can reenable the job.
+  #
+  # typescript:
+  #   name: Typescript compile check
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Setup Node
+  #       uses: actions/setup-node@v3
+  #       with:
+  #         node-version-file: '.nvmrc'
+  #         cache: 'yarn'
+  #     - name: Run typecript compile check
+  #       run: npx ts-compile-checker
+  #       working-directory: ./
```

Comment:
I think it is quite worrying if we do not compile our TypeScript as a part of our CI process.

`ts-compile-checker` might not work but the scope of this project (microservices etc.) also seems to expand beyond what is needed for `dpl-react`. Can we use a simpler solution instead?

It seems as if [`setup-node` already has a problem matcher for TypeScript](https://github.com/actions/setup-node/blob/main/.github/tsc.json) to running `tsc` might be sufficient.

Reply:
I don't follow what you are suggesting?
I agree in that we should have some ci ts compile checking.
We could have an entry:
`run: npx tsx`
is that what you mean?

Reply:
Found a solution: https://github.com/reload/dpl-react/pull/10/commits/9a6251992baf7f11c29c0f6a86e71aa308e37ded